lib/tasks/update_application_form_status.rake

```diff
@@ -0,0 +1,7 @@
+namespace :db do
+  desc "Update application status"
+  task update_status: :environment do
+    # update all application forms with 'initial assessment' status
+    forms.update_all(state: "assessment_in_progress")
```

Comment:
It doesn't look like a `forms` variable is defined here so I'm not sure the task will work at the moment.